tests/sdfg/work_depth_test.py

```diff
@@ -217,6 +219,17 @@ def test_work_depth(test_name):
         sdfg.apply_transformations(NestSDFG)
     if 'nested_maps' in test.name:
         sdfg.apply_transformations(MapExpansion)
+
+    # NOTE: Until the W/D Analysis is changed to make use of the new blocks, inline control flow for the analysis.
+    inliner = InlineControlFlowRegions()
+    inliner.no_inline_conditional = False
+    inliner.no_inline_loops = False
+    inliner.no_inline_function_call_regions = False
+    inliner.no_inline_named_regions = False
+    inliner.apply_pass(sdfg, {})
+    for sd in sdfg.all_sdfgs_recursive():
+        sd.using_experimental_blocks = False
```

Comment:
Currently there are still quite a few things that were constructed and are tested for the 'old' style - including some schedule tree things and one or two bespoke analysis passes. For everything except these analysis passes there were adaptations to work with the DaCe 2.0 style, but there is a massive code change pending to remove all of the legacy style code, including corresponding tests. I am not in favor of including this in this PR due to the ballooning effect of the line changes this will have, but as already mentioned in the description of the PR, there will be a subsequent PR following this one (which is already in the works), which removes all legacy code - and consequently also all need for this flag and all corresponding checks.